Resources/Prototypes/Recipes/Construction/structures.yml

```diff
@@ -110,3 +110,62 @@
   objectType: Structure
   placementMode: SnapgridCenter
   canRotate: false
+
+- type: construction
+  name: Firelock
+  id: Firelock
+  graph: Firelock
+  startNode: start
+  targetNode: Firelock
+  category: Structures
+  description: This is a firelock - it locks an area when a fire alarm in the area is triggered. Don't get squished!
+  icon:
+    sprite: Constructible/Structures/Doors/firelock.rsi
+    state: closed
+  objectType: Structure
+  placementMode: SnapgridCenter
+
+- type: construction
+  name: Table Frame
+  id: TableFrame
+  graph: Tables
+  startNode: start
+  targetNode: TableFrame
+  category: Structures
+  description: A square piece of metal standing on four metal legs. (What did you expect?)
```

Comment:
Given this is a table *frame* perhaps it should have a different description.

Reply:
~~Technically it's still a square piece of metal standing on four lags~~ Sure how about "A sturdy frame used in the construction of tables"